src/assets/Keycloak/ocis-with-keycloak.md

```diff
@@ -0,0 +1,161 @@
+---
+title: keycloak Server
+authorName: Amrita Shrestha
+authorAvatar: https://avatars.githubusercontent.com/u/54478846?s=100&v=4
+authorLink: https://github.com/amrita-shrestha
+createdAt: March 26, 2024
+tags: owncloud infinte scale, ocis, ocis-with-keycloak
+banner: https://blog.jankaritech.com/src/assets/keycloak/images/keycloak-with-ocis.png
+---
+
+[In previous blog](src/assets/keycloak/keycloak.md), we explored the topic of Keycloak. In this current post, our focus shifts to discussing the setup of OCIS with Keycloak.
+
+
+## Keycloak with ocis
+
+### Run services
+Before setting up `ocis with keycloak`, it's essential to deploy `keycloak and ocis`. This blog details the deployment of `ocis and keycloak` via `docker compose`.
+Using `Traefik`, `ocis and keycloak` has been served using certificates.
```

Comment:
how do the certificates serve ocis and keycloak?
Do you mean that you are using traefik to be able to access ocis and keycloak over secure TLS connections?